docs/eventing/broker/rabbitmq-broker/README.md

```diff
@@ -88,7 +106,11 @@ For more information about configuring the `RabbitmqCluster` CRD, see the
       name: <rabbitmq-broker-config-name>
     spec:
       rabbitmqClusterReference:
+        # if a local cluster is being used. name and connectionSecret cannot be set at the same time
```

Comment:
```suggestion
        # Configure name if a local cluster is being used.
```

Reply:
I deleted the _"name and connectionSecret cannot be set at the same time"_ part here. Maybe add this as a note later?